### PR TITLE
eyeD3.md: add page

### PR DESCRIPTION
**Pull request by sbrl**
I was going to create a page on composer, but I got distracted and did one for eyeD3 instead :P It's a tool for manipulating mp3 files which I use for my personal music collection, and I keep forgetting the syntax for setting the cover art :P

### CODE REVIEW

File: pages/linux/eyeD3.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# eyeD3
+
+> Manipulate mp3 files.
+
+- View information about an mp3 file:
+
+`eyeD3 {{Sounds.mp3}}`
```
Comment (agnivade) [inline pages/linux/eyeD3.md]: {{Sounds.mp3}} => {{filename.mp3}}

**Comment by sbrl** [comment]
@agnivade Good suggestions - is that better?

And yeah, it does _only_ take mp3 files, but if you are that way inclined you can give all your mp3 files a different extension and it won't particularly care.

**Comment by sbrl** [comment]
that new description reads much better :D

**Comment by agnivade** [comment]
Yes, I was talking about extensions. If that does not matter, then its not needed to put it outside the tokens. 

I made some changes in the description to make the tool's function a bit clearer. Let me know what you think of those.

**Comment by sbrl** [comment]
Ummm bump?

**Comment by agnivade** [comment]
I am waiting for @jeeftor or @waldyrious to lend their thoughts.

**Comment by sbrl** [comment]
Awesome, thanks @waldyrious :D

**Pull request merged**